### PR TITLE
Feature/df 1755 search by beruf

### PR DESCRIPTION
**Pull request by ganqqwerty**

### CODE REVIEW

File: alv-portal-ui/src/app/competence-catalog/ch-fiches/ch-fiches-overview/ch-fiches-overview.component.html [hunk 1]
```diff
@@ -23,10 +24,24 @@ <h2 class="pr-5 mb-4 d-flex">
...
-      <alv-input-field [alvControl]="query"
-                       label="portal.competence-catalog.ch-fiches.overview.search-label">
-      </alv-input-field>
+      <div class="row">
+        <div class="col-12 col-lg-6 mb-1 mb-lg-0">
+          <alv-input-field alvFormControlName="query"
+                           label="portal.competence-catalog.ch-fiches.overview.search-label">
+          </alv-input-field>
+        </div>
+        <div class="col-12 col-lg-6 mb-1 mb-lg-0">
```
Comment (cjpsycos) [inline alv-portal-ui/src/app/competence-catalog/ch-fiches/ch-fiches-overview/ch-fiches-overview.component.html]: first occupation than query field

File: alv-portal-ui/src/app/competence-catalog/competence-sets/competence-element-search-modal/competence-element-search-modal.component.ts [hunk 2]
```diff
@@ -67,6 +68,7 @@ export class CompetenceElementSearchModalComponent implements OnInit {
     return this.competenceElementRepository.search({
       page: 0,
       size: DEFAULT_PAGE_SIZE,
+      sort: DEFAULT_SORT.desc,
```
Comment (cjpsycos) [inline alv-portal-ui/src/app/competence-catalog/competence-sets/competence-element-search-modal/competence-element-search-modal.component.ts]: default should be alphabetically asc

File: alv-portal-ui/src/app/competence-catalog/ch-fiches/ch-fiches-overview/ch-fiches-overview.component.html (hunk 1 above)
Comment (emina-str) [reply alv-portal-ui/src/app/competence-catalog/ch-fiches/ch-fiches-overview/ch-fiches-overview.component.html]: fixed

File: alv-portal-ui/src/app/competence-catalog/competence-sets/competence-element-search-modal/competence-element-search-modal.component.ts (hunk 2 above)
Comment (emina-str) [reply alv-portal-ui/src/app/competence-catalog/competence-sets/competence-element-search-modal/competence-element-search-modal.component.ts]: fixed

**Pull request merged**